examples/vulkan/Vulkan.cpp

```diff
@@ -253,6 +269,7 @@ class VulkanExample
         if (vulkanAvailable) setupDraw();
         if (vulkanAvailable) setupSemaphores();
         if (vulkanAvailable) setupFences();
+        // clang-format on
```

Comment:
Would be reformatted IMHO as well. Either that or then the block below shouldn't be reformatted either 😄

Reply:
Good point. I'll remove that `// clang-format off` block. I mostly want to reserve our use of `// clang-format off` for tabular data.